tasks/go.py

```diff
@@ -21,14 +21,15 @@
 from .utils import get_build_flags
 
 # List of modules to ignore when running lint
-MODULE_WHITELIST = [
+MODULE_ALLOWLIST = [
```

Comment:
Can you do this name change (and the change in the printed message) in a separate PR?

Reply:
Sounds good, I will extract this into a separate PR.